src/Microsoft.AspNetCore.Hosting.Abstractions/IWebHost.cs

```diff
@@ -24,6 +26,13 @@ public interface IWebHost : IDisposable
         /// <summary>
         /// Starts listening on the configured addresses.
         /// </summary>
-        void Start();
+        Task StartAsync(CancellationToken cancellationToken);
+
+        /// <summary>
+        /// Attempt to gracefully stop the host.
+        /// </summary>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
+        Task StopAsync(CancellationToken cancellationToken);
```

Comment:
👍 Still, having a test that calls StopAsync and verifies that it is in any way different than Dispose would probably be nice since it's a new method on WebHost.